__tests__/search.test.ts

```diff
@@ -14,18 +14,25 @@ vi.mock(import("@/lib/config/goConfig"), async importOriginal => {
   }
 })
 
-export const facets = {
-  [FacetFieldEnum.Materialtypesgeneral]: {
+type Facets = {
+  [key in FacetFieldEnum]?: {
```

Comment:
I thought that a part of this PR was to make codegen use `as const` instead of enums. I know that the Graphql still have the native type `enum` but then it could be converted?

Reply:
The new generated enums will be a type of union like this: 

```
export type AccessTypeCodeEnum =
  | 'ONLINE'
  | 'PHYSICAL'
  | 'UNKNOWN';
```